src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const nav = document.querySelector('.gallery__list');
+const imgLarge = document.querySelector('.gallery__large-img');
+
+nav.addEventListener('click', (events) => {
```

Comment:
The parameter should be named `event` instead of `events` to match the common convention and avoid confusion.